packages/backend-core/package.json

```diff
@@ -39,6 +39,7 @@
     "joi": "17.6.0",
     "jsonwebtoken": "9.0.0",
     "koa-passport": "4.1.4",
+    "koa-pino-logger": "4.0.0",
```

Comment:
Is there a reason why we fix the npm versions?

Reply:
Just as a best practice for security mostly. We also had some problems with drift between services as well, where we want worker, server and account portal to be on the same version of some lib, but the locked version have diverged